packages/router/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "@eggjs/router",
+  "version": "4.0.0-beta.15",
+  "engines": {
+    "node": ">=22.18.0"
+  },
+  "publishConfig": {
```

Biome (2.1.2)

[error] The key publishConfig was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)